### PR TITLE
linters-added

### PR DESCRIPTION
**Pull request by IvanLudick**
@BregornOriginal I made changes to the readme file, added test.md file

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1,97 @@
...
\ No newline at end of file
+# HTML & CSS3 Course
+
+If you are not familiar with linters, read [root level README](../README.md).
+
+## Set-up GitHub Actions
+
+Please do the following **steps in this order**:
+
+1. In the first commit of your feature branch create a `.github/workflows` folder and add a copy of [`.github/workflows/linters.yml`](.github/workflows/linters.yml) to that folder.
```
Comment (BregornOriginal) [inline README.md]: - [ ] Remember that the objective of this lesson and from now on is to follow the next [README TEMPLATE OF MICROVERSE](https://github.com/microverseinc/readme-template). Kindly use that template and complete it with the information about this protect 💯 💯 💯

File: index.html [hunk 2]
```diff
@@ -0,0 +1,13 @@
+<!DOCTYPE html>
```
Comment (BregornOriginal) [inline index.html:1]: - [x] Kindly create a test.md file that includes the text as it's said on the requirements and add it to the `.gitignore` file

image

File: index.html [hunk 3]
```diff
@@ -0,0 +1,13 @@
...
+<html lang="en">
```
Comment (BregornOriginal) [inline index.html:2]: - [ ] Remember that when we create a Pull Request, the person that it's going to review your project needs to know on the first view the changes that you're doing on that PR. Kindly add a description of the changes that you are making on this PR, you can use the requirements of this activity for help 💯

**Review by BregornOriginal — CHANGES_REQUESTED** [review]
## Hi @IvanLudick 💯 💯 💯 

## Good job so far!
## There are some issues that you still need to work on to go to the next project but you are almost there!

one-step-closer-one-more

## Highlights

✅ Linters are passing
✅ Good use of GitHub Flow
✅ Good Pull Request description

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
- [OPTIONAL] **From now on remember to use meaningful commits. Instead of `Added` you have to say `Add`**


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by tufoinnkuo10 — APPROVED** [review]
## Status: Approved :heavy_check_mark:

Hi @IvanLudick 

Your project is complete! Great job for making the requested changes🥇. There is nothing else to say other than... it's time to merge it 🍾🚢 :shipit: . Congratulations!  🎉💯🌟 

## Good Points 👍
- Descriptive Pull Request
- good readme template
- No linter errors

<br>
Cheers..Happy coding!..and keep soaring higher! 💻 🍷 🚀 

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have 2 more limited reviews per this project. If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**